e2e/specs/math.spec.ts

```diff
@@ -14,50 +14,69 @@ const metaKey = process.platform === "darwin" ? "Meta" : "Control";
 test.beforeEach(async ({ page }) => {
   await page.goto("/subject-matter/learning-resource/new");
 
+  await page.waitForTimeout(300);
```

Comment:
Matte- og toolbar-specs påvirker hverandre av en eller annen grunn. Satt en timeout for å unngå det.